### PR TITLE
Create device_tracker.hitron_coda.markdown

### PR DESCRIPTION
**Pull request by arilotter**
**Description:**
Add docs for new device tracker (hitron CODA)

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#9682

### CODE REVIEW

File: source/_components/device_tracker.hitron_coda.markdown [hunk 1]
```diff
@@ -0,0 +1,36 @@
+---
+layout: page
+title: "Hitron CODA Routers"
+description: "Instructions how to integrate Hitron CODA Routers into Home Assistant."
```
Comment (frenck) [inline source/_components/device_tracker.hitron_coda.markdown]: **Spelling & Grammar**:
- `Instructions how to` -> `Instructions on how to`

File: source/_components/device_tracker.hitron_coda.markdown [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+device_tracker:
+  - platform: hitron_coda
+    host: !secret router_ip
+    username: !secret router_username
+    password: !secret router_password
+```
+
+Configuration variables:
+
+- **host** (*Required*): The IP address of your router, e.g. 192.168.0.1.
```
Comment (frenck) [inline source/_components/device_tracker.hitron_coda.markdown]: **Spelling & Grammar**:
- Missing comma after `e.g.` -> `your router, e.g., 192.168.0.1`

File: source/_components/device_tracker.hitron_coda.markdown [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+- **username** (*Required*): The username to login into the router (user should have read access to web interface of the router). Usually "cusadmin".
```
Comment (frenck) [inline source/_components/device_tracker.hitron_coda.markdown]: **Spelling & Grammar**:
- `to web interface` -> `to the web interface`

File: source/_components/device_tracker.hitron_coda.markdown [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+title: "Hitron CODA Routers"
+description: "Instructions on how to integrate Hitron CODA Routers into Home Assistant."
+date: 2017-10-03 15:40
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: keenetic.png
+ha_category: Presence Detection
+ha_release: 0.55
```
Comment (MartinHjelmare) [inline source/_components/device_tracker.hitron_coda.markdown]: 0.58

File: source/_components/device_tracker.hitron_coda.markdown [hunk 5]
```diff
@@ -0,0 +1,36 @@
+---
+layout: page
+title: "Hitron CODA Routers"
+description: "Instructions on how to integrate Hitron CODA Routers into Home Assistant."
+date: 2017-10-03 15:40
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: keenetic.png
```
Comment (MartinHjelmare) [inline source/_components/device_tracker.hitron_coda.markdown]: Stale logo.

File: source/_components/device_tracker.hitron_coda.markdown [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+```yaml
+# Example configuration.yaml entry
+device_tracker:
+  - platform: hitron_coda
+    host: !secret router_ip
+    username: !secret router_username
+    password: !secret router_password
+```
+
+Configuration variables:
```
Comment (MartinHjelmare) [inline source/_components/device_tracker.hitron_coda.markdown]: Use the new configuration tag.

https://home-assistant.io/developers/documentation/standards/#component-and-platform-pages

**Comment by frenck** [comment]
@arilotter Since the parent PR has been merged, we hope to be able to merge the documentation as well. Could you please take a look at the comments left behind by @MartinHjelmare? Thx 🏅

**Comment by frenck** [comment]
v0.58 is getting close and no response from the author. I will be updating this PR myself.

**Comment by arilotter** [comment]
Hey @frenck, I left a comment on the review because I was confused about how to implement the {% configuration %} tag. Thanks for updating the PR yourself!

**Comment by frenck** [comment]
@arilotter No problem! There is a little deadline, so that's why I did it.

**Pull request merged**